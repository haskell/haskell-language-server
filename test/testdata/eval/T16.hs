{-# LANGUAGE TypeApplications #-}
module T16 where

foo :: Show a => a -> String
foo = show

-- >>> :type  +d   40+ 2 
