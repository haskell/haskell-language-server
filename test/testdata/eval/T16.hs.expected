{-# LANGUAGE TypeApplications #-}
module T16 where

foo :: Show a => a -> String
foo = show

-- >>> :type  +d   40+ 2 
-- 40+ 2 :: Integer
